app/models/post.rb

```diff
@@ -78,6 +80,8 @@ class Post < ActiveRecord::Base
 
   SHORT_POST_CHARS ||= 1200
 
+  # TODO: Drop the data from these three custom fields,
```

Comment:
Can we add a "after n" version to this todo comment?

Reply:
I intend to drop the old data within a day or two of merging (once we're 100% confident the change won't need to be reverted). Just opened a draft PR here so we don't forget: #16594